render.js

```diff
@@ -0,0 +1,17 @@
+const Renderer = function () {
+    const renderPosts = function (posts) {
+        $('#posts').empty()
```

Comment:
Create multiple helper functions for this to separate the different concerns. At least 2 more functions.